src/AzSK.ADO/Framework/Core/SVT/SVTResourceResolver.ps1

```diff
@@ -1091,7 +1091,7 @@ class SVTResourceResolver: AzSKRoot {
                     # getting all the resources count
                     # and sending them to telemetry as well
                     $scanSource = [AzSKSettings]::GetInstance().GetScanSource(); # Disabling resource telemetry for SDL scan.
-                    if($this.IsAIEnabled -eq $true -and $scanSource -ne 'SDL') {
+                    if($env:fetchInventoryDetails -eq $true -and $this.IsAIEnabled -eq $true -and $scanSource -ne 'SDL') {
```

Comment:
Please follow same pattern for naming of env, Should start with caps

Can we  disable this by default and enable it based on env variable?